library/ramltojaxrs.raml

```diff
@@ -0,0 +1,14 @@
+#%RAML 1.0 Library
+annotationTypes:
+  types:
+    allowedTargets: [TypeDeclaration,API]
+    properties:
+      className?:
+        type: string
+      implementationClassName?:
+        type: string
+      usePrimitiveType?:
+        type: boolean
+      abstract?:
+        type: boolean
+      plugins?: any[]
```

Comment:
Can you add a new line at the end? GitHub and many editors/IDEs put a warning here because it violates the POSIX standard.

Reply:
Done